src/azure-cli/azure/cli/command_modules/profile/_validators.py

```diff
@@ -12,20 +12,20 @@
 def validate_tenant(cmd, namespace):
     """
     Make sure tenant is a GUID. If domain name is provided, resolve to GUID.
-    https://docs.microsoft.com/azure/active-directory/develop/v2-protocols-oidc#fetch-the-openid-connect-metadata-document
+    https://learn.microsoft.com/en-us/entra/identity-platform/v2-protocols-oidc#find-your-apps-openid-configuration-document-uri
```

Comment:
The other day I learn this trick from a Content Team writer. They said, most (if not all) of the learn.microsoft.com pages shall omit the `/en-us` part from the url, so that the website will automatically redirect the reader into a page written with their browser's preferred language.
```suggestion
    https://learn.microsoft.com/entra/identity-platform/v2-protocols-oidc#find-your-apps-openid-configuration-document-uri
```

Reply:
Thanks for the suggestion. Actually, I am well aware of this (omitting `en-us`). This is actually a requirement for customer-facing URLs. However, this URL here appears in code comment which is only for developers, so the English version should be the single source of truth.